waspc/src/Wasp/Generator/DbGenerator.hs

```diff
@@ -43,13 +50,32 @@ dbSchemaFileInDbRootDir = SP.castRel dbSchemaFileInDbTemplatesDir
 dbSchemaFileInProjectRootDir :: Path' (Rel ProjectRootDir) File'
 dbSchemaFileInProjectRootDir = dbRootDirInProjectRootDir </> dbSchemaFileInDbRootDir
 
+dbMigrationsDirInDbRootDir :: Path' (Rel DbRootDir) (Dir DbMigrationsDir)
```

Comment:
Aha, so you see, you named the DbMigrationsDir in the generated code here. So referring to my previous comment, it would make sense to also name it in the frontend of the compiler.

Reply:
Will do!